package.json

```diff
@@ -18,10 +18,15 @@
       "it",
       "expect",
       "cy",
-      "Cypress"
+      "Cypress",
+      "i18n",
+      "PaymentRequest",
+      "ApplePaySession",
```

Comment:
This looks like it adds 5 dependencies, are these supposed to be in this PR?

Reply:
EDIT - didn't look at context 🤦‍♂️

Reply:
😅